driver/pelican/pelican.go

```diff
@@ -223,10 +221,19 @@ func (pel *Pelican) GetStatus() (*PelicanStatus, error) {
 	if histResult.Success == 0 {
 		return nil, fmt.Errorf("Error retrieving thermostat status from %s: %s", respHist.Request.URL, histResult.Message)
 	}
+
 	if len(histResult.Records.History) == 0 {
-		return nil, fmt.Errorf("Error: No timestamp records found in past %v from %s", diff, pel.target)
+		//return nil, fmt.Errorf("Error: No timestamp records found in past %v from %s", diff, pel.target)
+		return nil, nil
 	}
+
+	// Converting string timeStamp to int64 format
 	match := histResult.Records.History[len(histResult.Records.History)-1]
+	timeString := match.TimeStamp + ":00Z"
```

Comment:
Are you sure you want to add the `00Z` suffix? That would imply that the timestamp is in UTC, but we know that Pelicans use their notion of local time,

Reply:
That's a good point. The issue i'm running into right now is that if I don't add the suffix, the time.Parse() function will throw a parsing error at me. In other words, the timestamp that the pelican returns doesn't include the seconds and timezone, so for me to utilize the time.Parse function, I have to concatenate the :00Z suffix.

I guess one approach I could take is hard-coding a hashmap linking IANA timezones with their respective +/- in hours relative to UTC. Therefore, when I construct the timeString variable, I could append the correct timezone suffix.

I think the big problem with that approach is that the map will be extremely large due to all of the IANA timezones out there. I will look into whether there's some API I can use instead of the map. However, this is the only alternative I could conjure off the top of my head. I'm very open to suggestions.